### ISSUE TITLE
New widgets tray has some problems in QA

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/MapStore2#8801]
## Description

Using arrows to switch to a widget to another sometime the widget disappear and clicking on the tray button it doesn't appear again.

## How to reproduce

- Open [this](https://qa-mapstore.geosolutionsgroup.com/mapstore/#/viewer/openlayers/10358) map in mobile
- Expand the widget clicking on the widget tray button
- Use one arrow

*Expected Result*

As it is currently working in DEV.

*Current Result*

![widget_tray.gif](https://images.zenhubusercontent.com/5c9ca89d6607b6793c66ec35/b2f87a2f-d0a8-4182-a719-997caebd74ca)

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

**Comment by alex-fko** [comment]
Just as I mentioned in #8800, this problem is actually caused by widgets fails to load because of the configuration format that is not yet supported on QA. Selected widget for one widget layout is stored in the state, so once user switched to the one cannot be rendered, click on tray will try to toggle this specific widget off and on, which, obviously, will result into nothing as widget cannot be rendered anyway.

TL;DR Not a bug, caused by shared backend between DEV and QA and new format of configuration for widgets in several entities (maps, dashboards) that were re-saved on dev.

**Comment by tdipisa** [comment]
@offtherailz this should be solved with https://github.com/geosolutions-it/MapStore2/issues/8800#issuecomment-1313555288
